client/client.go

```diff
@@ -244,10 +254,21 @@ func IsTwoFactorError(err error) bool {
 	return e.Kind == ErrorKindTwoFactorFailed || e.Kind == ErrorKindTwoFactorRequired
 }
 
+// OSRelease contains information about the system extracted from /etc/os-release.
+type OSRelease struct {
+	ID         string `json:"id"`
```

Comment:
I'm somewhat on edge about this but I'd love to consider using the original `/etc/os-release` spelling of those. So `ID`, `VERSION_ID`, `PRETTY_NAME` etc.